apps/src/lab2/levelEditors/aichatSettings/EditAichatSettings.tsx

```diff
@@ -88,13 +89,30 @@ const EditAichatSettings: React.FunctionComponent<{
     [aichatSettings, setAichatSettings, initialCustomizations]
   );
 
+  const setModelSelectionValues = useCallback(
+    (availableModelIds: string[], selectedModelId: string = '') => {
+      setAichatSettings({
+        ...aichatSettings,
+        availableModelIds: availableModelIds,
+        initialCustomizations: {
+          ...aichatSettings.initialCustomizations,
+          ['selectedModelId']:
```

Comment:
the key should probably just be `selectedModelId`

Reply:
Is latest what you meant?